packages/docs/page-config/ui-elements/layout/index.ts

```diff
@@ -0,0 +1,67 @@
+export default definePageConfig({
+  blocks: [
+    block.title('Layout'),
+    block.paragraph('Component is used for building App layout. It is based on CSS Grid and provides a simple API for building complex layouts.'),
+    block.paragraph(`
+VaLayout component is used in pair with [VaSidebar](/ui-elements/sidebar)[[target=_blank]], [VaNavbar](/ui-elements/navbar)[[target=_blank]], VaFooter components.
+    `),
+
+    block.example('Default', {
+      title: 'Default usage',
+      description: 'Default usage of Layout component.',
+    }),
+    block.example('AllSlots', {
+      title: 'All slots',
```

Comment:
![image](https://github.com/epicmaxco/vuestic-ui/assets/5394573/64bc19ea-2e8e-40ce-a818-a034cb50b677)

This example doesn't look too nice on mobile.